test/functional/multisignatures.js

```diff
@@ -118,7 +118,8 @@ describe('POST signatures/sign (regular account)', function () {
 				node.expect(res.body.transaction).to.have.property('id').to.equal(transaction.id);
 				var signature = node.lisk.multisignature.signTransaction(transaction, multisigAccount.password);
 				sendSignature(signature, transaction, function (err, res) {
-					node.expect(err).not.to.be.empty;
+					node.expect(res).to.have.property('success').to.not.be.ok;
+					node.expect(res).to.have.property('message').to.equal('Invalid signature body Expected type object but found type string');
 					done();
```

Comment:
Should be at least: `Invalid signature body - Expected type object but found type string`